be/test/exprs/array_functions_test.cpp

```diff
@@ -5640,4 +5640,176 @@ TEST_F(ArrayFunctionsTest, array_contains_seq) {
         EXPECT_EQ(1, result->get(2).get_int8());
     }
 }
+
+TEST_F(ArrayFunctionsTest, array_repeat_int) {
```

Comment:
Could you add sql-test in /test to cover this new function?

Reply:
I am adding test cases to sufficient coverage this feature.

Reply:
Test cases have been added.